src/app/services/rss-feed.service.ts

```diff
@@ -31,10 +31,23 @@ export class RssFeedService {
           return txt.value;
         }(item.querySelector('title').innerHTML);
 
+        let categoryTags = item.querySelectorAll('category');
+        let categories: string[] = [];
+        for (let j = 0; j < categoryTags.length; ++j) {
+          // Get the value of the category tag
+          let category = function (html: string) {
+            let txt = <HTMLTextAreaElement> document.createElement('textarea');
+            txt.innerHTML = html;
+            return txt.value;
+          }(categoryTags[j].innerHTML);
```

Comment:
Yeah, this should work.  Bit odd to have these wrapper anonymous-functions here ... but I guess this matches the style!  And maybe there's a reason for it.

Reply:
I remember when I did it for the title it was necessary, but I don't remember why.